ChangeLog.d/format-signedness.txt

```diff
@@ -0,0 +1,2 @@
+Changes
+   * add -Wformat-signedness when gcc > 5.
```

Comment:
That's not what this change does: it removes `-Wformat-signedness` with gcc 4.8–4.9, or said otherwise, it ensures that -`Wformat-signedness` is only used with gcc > 5.0.

Also the first letter of the sentence and the proper name GCC need to be capitalized.

Also you deserve credit in the changelog entry (unless you don't want it). Feel free to use your GitHub handle, your real name, or any other reasonable attribution.

```suggestion
   * Only pass -Wformat-signedness to versions of GCC that support it. Reported
     in #3478 and fix contributed in #3479 by okhowang.
```